cmake/BuildFlags.cmake

```diff
@@ -66,7 +66,11 @@ if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
 
   set(SYCL_OFFLINE_COMPILER_CG_OPTIONS ${SYCL_OFFLINE_COMPILER_CG_OPTIONS} "-options \"-cl-intel-enable-auto-large-GRF-mode\"")
   # Support PVC AOT only currently.
-  set(SYCL_OFFLINE_COMPILER_AOT_OPTIONS "-device pvc,xe-lpg")
+  if((DEFINED ENV{TORCH_XPU_ARCH_LIST}) AND NOT ("$ENV{TORCH_XPU_ARCH_LIST}" STREQUAL ""))
```

Comment:
> `# Support PVC AOT only currently.`

Comment on previous line is outdated after adding MTL support. And will be even more outdated after this change below.

Reply:
Will remove.